styles/styles.css

```diff
@@ -244,6 +243,26 @@ a {
 	background-color: #fafafa;
 }
 
+.btn-delete {
+	background-color: #f0f0f0!important;
```

Comment:
any particular reason for he !important part?

Reply:
No, it's not needed. I've forgotten to remove it. Thanks